src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 64 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.can invoke a single script asynchronously, but only in a new runspace
```powershell
Cannot convert argument "tasks", with value: "System.Object[]", for "WaitAll" to type "System.Threading.Tasks.Task": "Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Threading.Tasks.Task"."
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 27
27:                     [System.Threading.Tasks.Task]::WaitAll(@($r))

```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.handles terminating errors properly when invoked asynchronously in a new runspace
```powershell
Expected an exception, with FullyQualifiedErrorId 'AggregateException' to be thrown, but the FullyQualifiedErrorId was 'MethodArgumentConversionInvalidCastArgument'. from /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1:69 char:21
    +                     [System.Threading.Tasks.Task]::WaitAll(@($r))
    +                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 73
73:                 { $sb.Invoke() } | Should -Throw -ErrorId 'AggregateException'
```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.cannot invoke a single script asynchronously in a runspace that is busy
```powershell
Expected an exception, with FullyQualifiedErrorId 'AggregateException' to be thrown, but the FullyQualifiedErrorId was 'MethodArgumentConversionInvalidCastArgument'. from /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1:27 char:21
    +                     [System.Threading.Tasks.Task]::WaitAll(@($r))
    +                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 115
115:                     $err = { InvokeAsyncHelper -PowerShell $ps2 -Wait } | Should -Throw -ErrorId 'AggregateException' -PassThru
```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Multiple script tests.can invoke multiple scripts asynchronously
```powershell
Cannot convert argument "tasks", with value: "System.Object[]", for "WaitAll" to type "System.Threading.Tasks.Task": "Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Threading.Tasks.Task"."
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 149
149:                 [System.Threading.Tasks.Task]::WaitAll(@($r1, $r2))

```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - With input and output.can invoke multiple scripts asynchronously with input
```powershell
Cannot convert argument "tasks", with value: "System.Object[]", for "WaitAll" to type "System.Threading.Tasks.Task": "Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Threading.Tasks.Task"."
at <ScriptBlock>, /Users/runner/runners/2.158.0/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 183
183:                 [System.Threading.Tasks.Task]::WaitAll(@($r1, $r2))

```